scripts/tests/sanitycheck/test_testsuite_class.py

```diff
@@ -3,17 +3,20 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 # pylint: disable=line-too-long
-# pylint: disable=C0321
+# pylint: disable=multiple-statements
 '''
 This test file contains testcases for Testsuite class of sanitycheck
 '''
 import sys
 import os
+import csv
+import pytest
+from mock import call, patch, MagicMock
```

Comment:
This needs to be added to requirements*.txt somewhere, I had to install it after import failed.

Reply:
Addressed this in requirements-build-test.txt.